src/mixins/audit.mixin.ts

```diff
@@ -82,8 +88,11 @@ export function AuditRepositoryMixin<
           data =>
             new AuditLog({
               actedAt: new Date(),
-              // eslint-disable-next-line @typescript-eslint/no-explicit-any
-              actor: (user?.id as any).toString() ?? '0', //NOSONAR
+              actor:
+                options?.actorId ??
```

Comment:
This seems repeated, can't you move this to a function? that can be overridden?

Reply:
done